src/Game/Data/types.ts

```diff
@@ -0,0 +1,34 @@
+import { LocalizedContent } from "../../Components/Localization";
+
+// Properties beyond name are all optional at this time to allow for gradual migration
+// Mainly provided here as examples of what we could do with the data
+
+export interface Action {
+	name: string;
+	id?: number; // Intended for use with exporting to Amarantine
```

Comment:
```suggestion
	id?: number; // Intended for use with exporting to Tischel
```
this is the in-game ID, correct?

Reply:
Correct, further updated the comment to clarify.